packages/kit/src/lib/HookSystem.ts

```diff
@@ -176,24 +241,40 @@ export class HookSystem<
 		return hooks;
 	}
 
+	/**
+	 * Returns list of hooks for a given name
+	 */
+	hooksForName(name: string): RegisteredHook[] {
+		return this._registeredHooks[name] ?? [];
+	}
+
 	createScope<TExtraArgs extends unknown[] = never[]>(
 		owner: string,
 		extraArgs: [...TExtraArgs],
-		meta: Partial<RegisteredHookMeta> = {},
-	): CreateScopeReturnType<THookFns, TExtraArgs> {
+		scopeMeta: Partial<RegisteredHookMeta> = {},
```

Comment:
What is the purpose of providing a scopeMeta? Does this add default meta properties to hooks registered within the scope?

Reply:
None really, let's get rid of it ☺️